src/pages/Dashboard.tsx

```diff
@@ -1,9 +1,10 @@
 import CssBaseline from '@mui/joy/CssBaseline';
 import Box from '@mui/joy/Box';
 
-import Header from '../components/Header';
-import Footer from '../components/Footer';
+
```

Comment:
Please remove all spaces between imports, than use `organise imports` from VSC.

Reply:
Maybe organize import will be good option to add in hushy ? I don't know it is possible probably yes. Do you have some special config ?

Reply:
Yes, but use your imagination 😉